### PR TITLE
docs: basic weave types walkthrough in example notebook

### PR DESCRIPTION
**Pull request by staceysv**
Super minimal walkthrough of the basic types Number and String. 

Main questions:
1. **Explain Weave more generally/fundamentally first?** Shawn's walkthrough starts by explaining Weave nodes, local artifact file paths, weave.use(), etc—can we refer to this notebook/assume folks will read this first? Or should we start with these fundamentals, or perhaps a condensed version of them?
2. **Level of "official reference"/technical detail, or number of examples per type:** more/fancier language, any particular style/spec guides I should follow? am I missing anything on the types that folks will need to know? Happy to compose fancier ops, show a bunch of other numeric types, show an example of every op, etc—prioritizing brevity for now
3. **EE / Op suggestions**: the current list of op suggestions is too flat / has a lot of noise—I am inclined to group these into categories and only talk about the relevant ones (even though ecosystem noise/generics like "mnist" and "food101" for Number or "chat_anthropic" /"chat_openai" for String will show up—any way to organize/filter these?

Please see the Todo section for more detailed questions—e.g. chainable_ops regression, specific ops and their usage, whether we can clean up op suggestions to make basic ops easier to find/use—I don't think we need to fix all of these to merge, and wanted to flag somewhere.

### CODE REVIEW

File: examples/experimental/skip_test/primitive_types.ipynb [hunk 1]
```diff
@@ -0,0 +1,275 @@
+{
```
Comment (jlzhao27) [inline examples/experimental/skip_test/primitive_types.ipynb:1]: kinda hard to comment on specific cells in json, but i think some conversion examples would be helpful, can we convert from 1 type to another in weave?

Comment (staceysv) [reply examples/experimental/skip_test/primitive_types.ipynb:1]: great point, added a basic conversion example between String and Number!

**Comment by staceysv** [comment]
@jlzhao27 added a type conversion, leaning towards making this a more WIP/in-depth notebook and adding pieces that don't fit in the main walkthrough

**Comment by staceysv** [comment]
@jlzhao27 added an MVP markdown doc to match the notebook

File: examples/experimental/skip_test/basic_weave_types.ipynb [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+    "pi_num = pi_str.toNumber()\n",
+    "pi_num"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b3f26cd3",
+   "metadata": {},
+   "source": [
+    "# TODO\n",
```
Comment (scottire) [inline examples/experimental/skip_test/basic_weave_types.ipynb:265]: I suggest we remove the TODO part here.

**Review by scottire — CHANGES_REQUESTED** [review]

**Review by scottire — APPROVED** [review]

**Pull request merged**